2015/proposals/agarwal-rajat.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Add taxonomic name resolution to the EcoData Retriever to facilitate data science approaches to ecology
+
+## Abstract
+
+The EcoData Retriever is a Python based tool for automatically downloading, cleaning up, and restructuring ecological data. It does the data munging so that the users can focus on using the data.
+As in ecological and evolutionary data, the names of species are constantly being redefined - it is difficult to combine datasets. By automating reconciliation of different species names as part of the process of accessing the data in the first place it will become much easier to combine diverse datasets and in new and interesting ways.
+
+## Technical Details
+
+The project would consists of the following components:
+
+1. Querying the taxonomy name resolution service:
+APIs of the name resolution services have to be used to fetch the possible names using Python. The name of the species would be the input to the APIs and their possible names would be returned. This will be a separate module for the purpose of reusability.      
+
+2. Decision of the best name
+From the possible names returned from different services, an algorithm to determine the best name has to be developed. This would include giving different weights to parameters like reliability of source, consistency with other services and other parameters which need to be thought of. 
+
+3. Updating the database
+Once the decision about the best name is made, this component will update the data model and the user interface with the best name and its location.
```

Comment:
I'm a little confused by what you mean here. You mention updating the database, changing the data model, and the user interface. I think of these as three separate components, so it would be good to be clearer about precisely what you're thinking here.